src/main/java/games/strategy/engine/data/GameDataMemento.java

```diff
@@ -0,0 +1,136 @@
+package games.strategy.engine.data;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+
+import games.strategy.util.Version;
+import games.strategy.util.memento.MementoExporter;
+import games.strategy.util.memento.MementoImportException;
+import games.strategy.util.memento.MementoImporter;
+import games.strategy.util.memento.PropertyBagMementoExporter;
+import games.strategy.util.memento.PropertyBagMementoImporter;
+
+/**
+ * Provides factory methods for creating objects that can import mementos to and export mementos from game data.
+ */
+public final class GameDataMemento {
+  @VisibleForTesting
+  static final String SCHEMA_ID = "application/x.triplea.game-data-memento";
```

Comment:
Can this be made into an enum value? I think the strong typing over a string would be useful to constrain these values. Overall I do think this is an enum since not any string is a schema id.